training/model_evaluation/environments/context/Dockerfile

```
@@ -0,0 +1,44 @@
+FROM mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20220516.v1
```

See https://github.com/Azure/azureml-assets/wiki/Environments#template-tags
```suggestion
FROM mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:{{latest-image-tag}}
```